CharacterMap/CharacterMap/Views/FontMapView.xaml

```diff
@@ -93,6 +104,43 @@
                         TextTrimming="CharacterEllipsis" />
                 </Grid>
             </Button>
+
+            <AutoSuggestBox
+                x:Name="SearchBoxUnicode"
```

Comment:
Name and Uid need to be changed to "SearchBox", because this is not only Unicode now.